addons/l10n_in/views/report_invoice.xml

```diff
@@ -64,4 +68,10 @@
         </xpath>
     </template>
 
+    <template id="place_of_supply">
+        <div t-if="o.country_code == 'IN' and o.l10n_in_state_id">
```

Comment:
not sure it's needed to check the fiscal country code here since this template is called only if `_get_name_invoice_report` returned its name, which occurs only if the fiscal country code is 'IN'